README.md

```diff
@@ -29,27 +29,12 @@ Permissions:
 
 # Quick start guide:
 
-1. Ensure there are existing channels in Bloom Collective, and Bloom Studio Discord servers with the channel names set in constants.py. The bot will determine the required channel IDs / Forum Channel IDs from the channel names in it's respective server.
+1. To run in prod set env var "ENV" to prod. Ensure there are existing channels in Bloom Studio Discord servers with the channel names set in constants.py. The bot will determine the required channel IDs / Forum Channel IDs from the channel names in it's respective server."
```

Comment:
some grammar fixes:

To run in production mode, set env var "ENV" to "prod". Ensure there are existing channels in Bloom Studio Discord servers with the channel names set in `constants.py`. The bot will determine the required channel IDs / Forum Channel IDs from the channel names in its respective server.